packages/apps/plugins/plugin-markdown/src/props.ts

```diff
@@ -15,8 +15,8 @@ export const isMarkdown = (datum: unknown): datum is ComposerModel =>
   datum && typeof datum === 'object'
     ? 'id' in datum &&
       typeof datum.id === 'string' &&
-      'content' in datum &&
-      (typeof datum.content === 'string' || datum.content instanceof YText)
+      (typeof (datum as { [key: string]: any }).content === 'string' ||
```

Comment:
Why change this to casting?

Reply:
The `in` operator doesn’t work for the `Proxy`s of `TypedObject`s we may sometimes see here, so trying to `get` the property with dot notation seemed like a reasonable workaround, though I’d defer to any better pattern.